modules/claims_api/app/controllers/claims_api/v2/veterans/intent_to_file_controller.rb

```diff
@@ -40,17 +38,10 @@ def type
         def submit
           validate_request!(ClaimsApi::V2::ParamsValidation::IntentToFile)
 
-          type = ClaimsApi::IntentToFile::ITF_TYPES_TO_BGS_TYPES[params[:type].downcase]
-          participant_claimant_id = params[:participant_claimant_id] || target_veteran.participant_id
-          participant_vet_id = params[:participant_vet_id] || target_veteran.participant_id
-
-          bgs_response = bgs_service.intent_to_file.insert_intent_to_file(
-            intent_to_file_type_code: type,
-            participant_claimant_id: participant_claimant_id,
-            participant_vet_id: participant_vet_id,
-            received_date: Time.zone.now.strftime('%Y-%m-%dT%H:%M:%S%:z'),
-            submitter_application_icn_type_code: ClaimsApi::IntentToFile::SUBMITTER_CODE
-          )
+          type = itf_types[params[:type].downcase]
+          options = intent_to_file_options(type)
+          check_for_invalid_survivor_submission(options) if type == 'S'
```

Comment:
I was debating moving these around if we remove the `participantVetId` but now I can see why we need the `options` for the `ssn`...